webbpsf/webbpsf_core.py

```diff
@@ -885,25 +885,15 @@ def _get_telescope_pupil_and_aberrations(self):
                                 "that type: {}".format(type(self.pupil)))
             # ---- apply pupil intensity and OPD to the optical model
 
-            # TODO - more flexibly be smart about if the pupil size works for the LOM or not...
-
-            if 'npix1024' in pupil_transmission:
-                # The linear model is limited to require 1024 pixels right now, so in this case (the default)
-                # we can use that:
-                pupil_optic = opds.OTE_Linear_Model_WSS(
-                    name='{} Entrance Pupil'.format(self.telescope),
-                    transmission=pupil_transmission,
-                    opd=opd_map,
-                    v2v3=self._tel_coords()
-                )
-            else:
```

Comment:
Yay for getting rid of this if/else case for not being able to support other sizes of OTE model!